specs/connector.yaml

```diff
@@ -470,6 +470,12 @@ definitions:
       Combination of the cloud platform and location to provision this
       resource within.
     pattern: ^([a-z0-9][a-z0-9\-_]{1,63})::([a-z0-9][a-z0-9\-_]{1,63})$
+  ImportCode:
```

Comment:
@tim-speed Do we need to return this info from the connector? Is there a reason for providers to retrieve the code after importing a resource?

Reply:
Hrmm nope, and we wont be storing it. Should just be in the Provider spec.

Reply:
We wont be storing it on the resource at least. It will still be in the provision operation

Reply:
HRMmmm or wait, maybe we should store it on the resource as an indicator that the resource was imported?
What do you think @luizbranco  ? or should we just make it a boolean on the resource

Reply:
I'm fine saving the code with the resource and for the client we turn into a bool. What do you think? But I don't think we need to change the connector, providers should already have the mapping on their side.

Reply:
:+1: works for me.